src/containers/edit-profile/add-professional-category-modal/AddProfessionalCategoryModal.styles.ts

```diff
@@ -0,0 +1,47 @@
+import palette from '~/styles/app-theme/app.pallete'
+import { TypographyVariantEnum } from '~/types'
+
+const { Subtitle1, Body2 } = TypographyVariantEnum
+
+const titleWithDescription = {
+  wrapper: { textAlign: 'left' },
+  title: { typography: Subtitle1 },
+  description: { typography: Body2, color: 'primary.500' }
+}
+
+export const styles = {
+  root: { p: 5 },
+  formWrapper: {
+    mt: 2,
+    my: 3,
```

Comment:
can you just define a single m property, and we preffer to use px

Reply:
I will not change it because of our latest expert's words, we should use as less fixed pixel values as possible. Also we are going to migrate to design system in the nearest future, so we will use paddings from the system.

Reply:
ok, but, I still think that it is better to use one margin property instead of two for now, even if it will be changed in the future

Reply:
Fixed

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/b79ca416-295e-4bef-b1d9-d92314985a42)